sheet4DiegoXavi.tex

```diff
@@ -0,0 +1,145 @@
+\documentclass[12pt,a4paper]{article}
+
+\usepackage{enumerate}
+\usepackage{graphicx}
+\usepackage{float}
+\usepackage[a4paper, margin=3.5cm]{geometry}
+\setlength{\parindent}{2em} 
+\setlength{\parskip}{1em} 
+\usepackage{amsfonts}
+\usepackage{amssymb}
+\usepackage{amsmath}
+\usepackage{comment}
+
+\newcommand{\cA}{\mathcal{A}}
+\newcommand{\cS}{\mathcal{S}}
+\newcommand{\RR}{\mathbb{R}}
+\newcommand{\CC}{\mathbbm{C}}
+\newcommand{\vol}{\text{vol}}
+\newcommand{\id}{\text{id}}
+
+\begin{document}
+
+\noindent
+\textbf{UPC MAMME / Discrete Geometry 2018}
+
+\noindent
+\textbf{Sheet 4}
+
+\noindent
+\textbf{Diego Acedo and Xavier Gombau}
+
+
+\subsubsection*{Solution 1}
+
+There is one thing which we do not find entirely clear in this problem. 
+The question whether we are given $P$ or not.
+
+If $P$ is not given, we have discussed with other colleagues the problem, and we have come to the conclusion that the lower faces of $Q^c$ are not determined from only the linear map $\pi : \RR^p \longrightarrow \RR^{q+1} $, and so it is not possible to give an algorithm to find such faces. 
+
+% Posibilidad de explicarlo mejor
```

Comment:
Well, yes, you should; however, I've seen the other solutions, so I guess I can spare you. But still, be aware that you haven't written down what you wanted to express, nor have you provided a clear and explicit link to the relevant information you're omitting